File: LeetCode75/Array-String/kidsWithGreatestCandies.py
#my solution:
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        """
        :type candies: List[int]
        :type extraCandies: int
        :rtype: List[bool]
        """
        maxCandies = []
        maxVal = max(candies)
        for candy in candies:
            if candy + extraCandies >= maxVal:
                maxCandies.append(True)
            else: maxCandies.append(False)
    
        return maxCandies
        

File: LeetCode75/Array-String/kidsWithGreatestCandies.js
//my solution:
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function (candies, extraCandies) {
  let maxCandies = [];
  const maxVal = Math.max(...candies);
  for (let candy of candies) {
    if (candy + extraCandies >= maxVal) {
      maxCandies.push(true);
    } else {
      maxCandies.push(false);
    }
  }
  return maxCandies;
};
